### PR TITLE
Resolves #1256, resolves #1258 - fix for undoing an object3d removal in subgraph

### LINKED ISSUE #1256: Crash after CMD-X + Undo using Gaze In plugin
Reproduce:
-Open this graph: http://vizor.io/fthr/cut-paste-crash/edit
-Inside the "3D model" subgraph, select all (CMD-A)
-Then cut all (CMD-Z)
-Then press CTRL-Z for undo.

Check console for crash.

After some tries, I figured out this happens when Gaze In plugin is present, and it has a 3D object selected in the dropdown (this doesn't happen if it says "select object".

### LINKED ISSUE #1258: Objects disappear after selecting all, cutting, and undoing
Reproduce:
-Load the Flamingo default file
-Go inside the Floor subgraph
-Select all (cmd-a) then cut all (cmd-x)
-Floor has turned to white (expected: floor would disappear).
-Press cmd-z to undo.
-Floor now disappears (expected: floor would come back).

The plugins are now all connected again, but floor can't be brought back.

This same thing can be reproduced with the 3D model subgraph (flamingo) in a similar way.

### PR DESCRIPTION
**Pull request by kkananen**
output proxy's arrayness was set incorrectly on undo cut

### CODE REVIEW

**Comment by kkananen** [comment]
Note this also does a require("three.js") in helpers.js, bringing in the entire three.js (creating the boilerplate for this test became too tedious)

File: browser/scripts/application.js [hunk 1]
```diff
@@ -2071,7 +2071,9 @@ Application.prototype.start = function() {
 Application.prototype.onCoreReady = function(loadGraphUrl) {
 	var that = this
 
-	E2.ui.init(E2);
+	if (E2.ui.init) {
+		E2.ui.init(E2);
```
Comment (kschzt) [inline browser/scripts/application.js]: I would rather mock this in the test than test for something in `E2.ui` here

File: browser/plugins/three_mesh.plugin.js [hunk 2]
```diff
@@ -50,7 +50,7 @@
 
 		if (this.object3d) {
 			this.object3d.traverse(function(n) {
-				if (n instanceof THREE.MorphAnimMesh) {
+				if (THREE.MorphAnimMesh && n instanceof THREE.MorphAnimMesh) {
```
Comment (kschzt) [inline browser/plugins/three_mesh.plugin.js]: should be safe to assume presence of THREE.MorphAnimMesh?

**Comment by kschzt** [comment]
lgtm

**Pull request merged**